README.adoc

```diff
@@ -152,7 +152,14 @@ Similarly, navigate to `{ping-api}/name-service` and observe a response with the
 
 == Triggering restart in name microservice
 
-An endpoint has been provided under the `name` microservice to set it to an unhealthy state in the health check. The unhealthy state will cause the readiness and liveness probe to fail. Use the `curl` command to invoke this endpoint by making a POST request to `{name-api}/unhealthy` -- if `curl` is unavailable then use https://www.getpostman.com/[Postman]. Run the following command to view the state of the pods:
+An endpoint has been provided under the `name` microservice to set it to an unhealthy state in the health check. The unhealthy state will cause the readiness and liveness probe to fail. Use the `curl` command to invoke this endpoint by making a POST request to `{name-api}/unhealthy` -- if `curl` is unavailable then use https://www.getpostman.com/[Postman].
```

Comment:
We can suggest postman as a potential tool to use.  Users can use anything that they like.